### PR TITLE
feat: Submission of care order (FPLA-2313)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-2313

### CODE REVIEW

File: ccd-definition/CaseField/CareSupervision/caseField.json [hunk 1]
```diff
@@ -2012,5 +2012,14 @@
     "FieldType": "Text",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "otherOrderType",
+    "Label": " ",
+    "FieldType": "Text",
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseField/CareSupervision/caseField.json:2021]: probably would be better to use YesOrNo type to give a clue what kind of values are possible

File: ccd-definition/CaseEvent/CareSupervision/MultiState.json [hunk 2]
```diff
@@ -312,7 +313,8 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/update-task-list/submitted"
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/update-task-list/submitted",
+    "EventEnablingCondition": "otherOrderType=\"NO\""
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEvent/CareSupervision/MultiState.json:317]: hmm, does not looks right, unless i missed something. Flag otherOrderType is set only on one particular event. Before event is saved, then  otherOrderType is unspecified and events wont be visible (I think they should). Should it not be 'otherOrderType!=\"YES\"' ?

Comment (SomeshRA) [reply ccd-definition/CaseEvent/CareSupervision/MultiState.json]: Changed it to  'otherOrderType!="YES"'

**Comment by tomaszpowroznik** [comment]
looks ok

**Review by SomeshRA — DISMISSED** [review]
LGTM

**Review by nraghu32 — APPROVED** [review]

**Pull request merged**